.github/actions/build-deb-package/action.yml

```
@@ -18,26 +18,30 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Check out repository
-      uses: actions/checkout@v4
-
     - name: Build Docker Image
-      run: docker build -f ${{ inputs.dockerfile_path }} -t ${{ inputs.docker_image }} .
+      run: docker build --pull -f ${{ inputs.dockerfile_path }} -t ice-deb-package-builder .
       shell: bash
 
     - name: Run Package Build with Mounted Source
       run: |
-        mkdir -p output
+        mkdir -p "$HOME/build"
         docker run --rm \
-          -v $PWD/ice:/workspace/ice \
-          -v $PWD/output:/workspace/build \
+          -v "$GITHUB_WORKSPACE:/workspace/ice" \
+          -v "$HOME/build:/workspace/build" \
           -e ICE_VERSION=${{ inputs.ice_version }} \
           -e DEB_BUILD_OPTIONS="${{ inputs.deb_build_options }}" \
-          ice-debian-package-builder
+          \ ice-deb-package-builder /workspace/ice/packaging/dpkg/build-package.sh
```

I don't t think this `\` is necessary.
```suggestion
          ice-deb-package-builder /workspace/ice/packaging/dpkg/build-package.sh
```